File: problems/1128-number-of-equivalent-domino-pairs/solution.java
class Solution {
    public int numEquivDominoPairs(int[][] dominoes) {
        int[] count = new int[100];
        int ans = 0;
        for (int[] domino : dominoes) {
            int a = Math.min(domino[0], domino[1]);
            int b = Math.max(domino[0], domino[1]);
            int val = a * 10 + b;
            ans += count[val];
            count[val]++;
        }
        return ans;
    }
}

File: problems/1128-number-of-equivalent-domino-pairs/solution.py
class Solution:
    def numEquivDominoPairs(self, dominoes: List[List[int]]) -> int:
        count = {}
        ans = 0
        for domino in dominoes:
            domino.sort()
            key = tuple(domino)
            if key in count:
                ans += count[key]
                count[key] += 1
            else:
                count[key] = 1
        return ans